doc/modules/changes.h

```diff
@@ -201,6 +201,12 @@
  * the log averaging will average 10^23 and 10^21 to 10^22.
  * <br>
  * (Shangxin Liu, 2015/08/09)
+ *
+ * <li> New: There is now a new initial condition in which the temperature field is perturbed 
+ * following the SAVANI shear wave velocity model by Auer et al., 2014. The data were 
+ * downloaded from http://n.ethz.ch/~auerl/research.html .
+ * <br>
+ * (Shangxin Liu, 2015/08/20)
```

Comment:
These entries are sorted in reverse order, i.e., newest on top. Can you change this?